### ISSUE TITLE
Add aria-live to Notifications

### ISSUE THREAD

**Issue by anthonydillon** [canonical/maas-ui#3941]
We need to make sure we add an appropriate aria-live to notifications (especially errors) which is missing in the legacy implementation.

**Comment by petermakowski** [comment]
A few notes on this.

`aria-live` may need to be added to [NotificationList](https://github.com/canonical-web-and-design/maas-ui/tree/main/src/app/base/components/NotificationList/NotificationList.tsx) component instead of each `Notification`.

We're often rendering Notifications conditionally and `aria-live` needs to be present on a node before there are updates within in in order for it to announce the text correctly.

Alternatively we can do some trickery with delayed rendering so that it happens behind the scenes. In any case, needs more investigation.

**Comment by petermakowski** [comment]
This is now tracked on JIRA: https://warthogs.atlassian.net/browse/MAASENG-2895
